JitsiMeetJS.ts

```diff
@@ -421,6 +421,16 @@ export default {
             });
     },
 
+    /**
+     * Manually create JitsiLocalTrack's from the provided track info, by exposing the RTC method
+     *
+     * @param {Array<Object>} tracksInfo - array of track information
```

Comment:
Can you please document this?

Reply:
One more thing: could we generate the sourceId internally?

Reply:
Sure! Ill do those changes, for sourceId, would a random number be sufficient? Or should it follow a certain pattern?

Reply:
I have just added everything, except the bit to generate source ID. is the general structure okay?

Added some tests too